### DISCUSSION TITLE
Companion class generated name is in plural form, is it intended?

### DISCUSSION THREAD

**Discussion by givip** [simolus3/drift#1153] [Q&A] [answered]
Table class: Humans
Generated class: Human
Generated companion class: Human**s**Companion

I would expect: **HumanCompanion**

**Comment by kuhnroyal** [comment, marked as answer]
Companions by default use the table name as base, not the entity name. You can try playing around with the `use_data_class_name_for_companions` builder option. https://moor.simonbinder.eu/docs/advanced-features/builder_options/
